### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

File: calibration/NUTNRB/CGINS-NUTNRB-01063__20221212.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...28, 275.09, 275.9, 276.72, 277.53, 278.34, 279.15, 279.96, 280.77, 281.58, 282.39, 283.2, 284.01, 284.82, 285.63, 286.44, 287.26, 288.07, 288.88, 289.69, 290.5, 291.31, 292.12, 292.93, 293.74, 294.55, 295.36, 296.17, 296.98, 297.79, 298.6, 299.41, 300.22, 301.03, 301.84, 302.65, 303.46, 304.27, 305.08, 305.89, 306.7, 307.51, 308.31, 309.12, 309.93, 310.74, 311.55, 312.36, 313.17, 313.98, 314.78, 315.59, 316.4, 317.21, 318.02, 318.83, 319.63, 320.44, 321.25, 322.06, 322.86, 323.67, 324.48, 325.29, 326.09, 326.9, 327.71, 328.51, 329.32, 330.12, 330.93, 331.74, 332.54, 333.35, 334.15, 334.96, 335.77, 336.57, 337.38, 338.18, 338.99, 339.79, 340.59, 341.4, 342.2, 343.01, 343.81, 344.61, 345.42, 346.22, 347.02, 347.83, 348.63, 349.43, 350.23, 351.04, 351.84, 352.64, 353.44, 354.24, 355.05, 355.85, 356.65, 357.45, 358.25, 359.05, 359.85, 360.65, 361.45, 362.25, 363.05, 363.85, 364.65, 365.45, 366.24, 367.04, 367.84, 368.64, 369.44, 370.23, 371.03, 371.83, 372.62, 373.42, 374.22, 375.01, 375.81, 376.6, 377.4, 378.2, 378.99, 379.78, 380.58, 381.37, 382.17, 382.96, 383.75, 384.55, 385.34, 386.13, 386.93, 387.72, 388.51, 389.3, 390.09, 390.88, 391.67, 392.46, 393.25, 394.04, 394.83, 395.62]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01063__20221212.csv:8]: spot checked - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20221212.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...9, 272.82, 273.64, 274.46, 275.28, 276.1, 276.93, 277.75, 278.57, 279.39, 280.21, 281.04, 281.86, 282.68, 283.5, 284.32, 285.15, 285.97, 286.79, 287.61, 288.43, 289.26, 290.08, 290.9, 291.72, 292.54, 293.37, 294.19, 295.01, 295.83, 296.65, 297.47, 298.3, 299.12, 299.94, 300.76, 301.58, 302.4, 303.22, 304.05, 304.87, 305.69, 306.51, 307.33, 308.15, 308.97, 309.79, 310.61, 311.43, 312.26, 313.08, 313.9, 314.72, 315.54, 316.36, 317.18, 318, 318.82, 319.64, 320.46, 321.28, 322.1, 322.92, 323.74, 324.55, 325.37, 326.19, 327.01, 327.83, 328.65, 329.47, 330.29, 331.1, 331.92, 332.74, 333.56, 334.38, 335.19, 336.01, 336.83, 337.65, 338.46, 339.28, 340.1, 340.92, 341.73, 342.55, 343.37, 344.18, 345, 345.81, 346.63, 347.44, 348.26, 349.08, 349.89, 350.71, 351.52, 352.34, 353.15, 353.96, 354.78, 355.59, 356.41, 357.22, 358.03, 358.85, 359.66, 360.47, 361.28, 362.1, 362.91, 363.72, 364.53, 365.34, 366.16, 366.97, 367.78, 368.59, 369.4, 370.21, 371.02, 371.83, 372.64, 373.45, 374.26, 375.07, 375.88, 376.69, 377.49, 378.3, 379.11, 379.92, 380.73, 381.53, 382.34, 383.15, 383.95, 384.76, 385.57, 386.37, 387.18, 387.98, 388.79, 389.59, 390.4, 391.2, 392.01, 392.81, 393.61, 394.42, 395.22, 396.02]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01091__20221212.csv:8]: spot checked - looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01094__20221212.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
..., 271.85, 272.66, 273.47, 274.27, 275.08, 275.89, 276.7, 277.51, 278.32, 279.13, 279.94, 280.75, 281.56, 282.37, 283.18, 283.99, 284.8, 285.61, 286.42, 287.23, 288.03, 288.84, 289.65, 290.46, 291.27, 292.08, 292.89, 293.7, 294.51, 295.32, 296.12, 296.93, 297.74, 298.55, 299.36, 300.17, 300.98, 301.78, 302.59, 303.4, 304.21, 305.02, 305.82, 306.63, 307.44, 308.25, 309.06, 309.86, 310.67, 311.48, 312.29, 313.09, 313.9, 314.71, 315.51, 316.32, 317.13, 317.93, 318.74, 319.55, 320.35, 321.16, 321.97, 322.77, 323.58, 324.38, 325.19, 326, 326.8, 327.61, 328.41, 329.22, 330.02, 330.83, 331.63, 332.44, 333.24, 334.04, 334.85, 335.65, 336.46, 337.26, 338.06, 338.87, 339.67, 340.47, 341.28, 342.08, 342.88, 343.68, 344.49, 345.29, 346.09, 346.89, 347.69, 348.5, 349.3, 350.1, 350.9, 351.7, 352.5, 353.3, 354.1, 354.9, 355.7, 356.5, 357.3, 358.1, 358.9, 359.7, 360.5, 361.29, 362.09, 362.89, 363.69, 364.49, 365.28, 366.08, 366.88, 367.67, 368.47, 369.27, 370.06, 370.86, 371.66, 372.45, 373.25, 374.04, 374.84, 375.63, 376.42, 377.22, 378.01, 378.81, 379.6, 380.39, 381.19, 381.98, 382.77, 383.56, 384.35, 385.15, 385.94, 386.73, 387.52, 388.31, 389.1, 389.89, 390.68, 391.47, 392.26, 393.05, 393.84]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01094__20221212.csv:8]: spot checked - looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00660__20221212.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
...273.42, 274.23, 275.03, 275.83, 276.63, 277.44, 278.24, 279.04, 279.84, 280.65, 281.45, 282.25, 283.05, 283.85, 284.66, 285.46, 286.26, 287.06, 287.87, 288.67, 289.47, 290.27, 291.08, 291.88, 292.68, 293.48, 294.29, 295.09, 295.89, 296.69, 297.49, 298.3, 299.1, 299.9, 300.7, 301.51, 302.31, 303.11, 303.91, 304.71, 305.51, 306.32, 307.12, 307.92, 308.72, 309.52, 310.32, 311.13, 311.93, 312.73, 313.53, 314.33, 315.13, 315.93, 316.73, 317.53, 318.34, 319.14, 319.94, 320.74, 321.54, 322.34, 323.14, 323.94, 324.74, 325.54, 326.34, 327.14, 327.94, 328.74, 329.54, 330.34, 331.14, 331.94, 332.74, 333.53, 334.33, 335.13, 335.93, 336.73, 337.53, 338.33, 339.12, 339.92, 340.72, 341.52, 342.32, 343.11, 343.91, 344.71, 345.51, 346.3, 347.1, 347.9, 348.69, 349.49, 350.29, 351.08, 351.88, 352.67, 353.47, 354.26, 355.06, 355.85, 356.65, 357.44, 358.24, 359.03, 359.83, 360.62, 361.42, 362.21, 363, 363.8, 364.59, 365.38, 366.18, 366.97, 367.76, 368.56, 369.35, 370.14, 370.93, 371.72, 372.51, 373.31, 374.1, 374.89, 375.68, 376.47, 377.26, 378.05, 378.84, 379.63, 380.42, 381.21, 382, 382.78, 383.57, 384.36, 385.15, 385.94, 386.72, 387.51, 388.3, 389.09, 389.87, 390.66, 391.44, 392.23, 393.02, 393.8]",
```
Comment (cdobs) [inline calibration/NUTNRM/CGINS-NUTNRM-00660__20221212.csv:8]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00272__20220919.csv [hunk 5]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+272,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347] ", source_file DI7272F.cal
+272,CC_offset,"[2147749032.4, 2148054906.3, 2147401286.4, 2147024544.4, 2147165457.9, 2147404069.3, 2147682273.1] ",
+272,CC_scale,"[2.17370243425e-007, 2.04940251512e-007, 2.02490483992e-007, 1.95876419937e-007, 2.12584590083e-007, 2.04889652444e-007, 2.15449972621e-007] ",
```
Comment (cdobs) [inline calibration/SPKIRB/CGINS-SPKIRB-00272__20220919.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00300__20220919.csv [hunk 6]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+300,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347]", source_file DI7300D.cal
+300,CC_offset,"[2147414743.0, 2147829905.6, 2147304692.6, 2147166042.3, 2147849853.8, 2147310061.2, 2147600132.5]",
+300,CC_scale,"[2.15543103666e-007, 2.04252587953e-007, 2.20553043322e-007, 2.12963413931e-007, 2.10820535126e-007, 2.03439292842e-007, 1.8779175822e-007]",
```
Comment (cdobs) [inline calibration/SPKIRB/CGINS-SPKIRB-00300__20220919.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00302__20220919.csv [hunk 7]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+302,CC_immersion_factor,"[1.368, 1.410, 1.365, 1.354, 1.372, 1.322, 1.347]", source_file DI7302B.cal
+302,CC_offset,"[2147954716.2, 2147136058.3, 2147392669.5, 2147716353.3, 2147337963.8, 2147900949.4, 2147960328.9]",
+302,CC_scale,"[2.10118171154e-007, 1.9772945443e-007, 2.05809933789e-007, 2.03728337726e-007, 1.89536101027e-007, 2.18511598011e-007, 2.12856146056e-007]",
```
Comment (cdobs) [inline calibration/SPKIRB/CGINS-SPKIRB-00302__20220919.csv:4]: looks good

**Review by cdobs — APPROVED** [review]
all look good to me

File: calibration/NUTNRB/CGINS-NUTNRB-01091__20221212.csv (hunk 2 above)
Comment (batryn) [reply calibration/NUTNRB/CGINS-NUTNRB-01091__20221212.csv:8]: spot checked - looks good :)

File: calibration/NUTNRB/CGINS-NUTNRB-01094__20221212.csv (hunk 3 above)
Comment (batryn) [reply calibration/NUTNRB/CGINS-NUTNRB-01094__20221212.csv:8]: spot checked - looks good :)

File: calibration/NUTNRM/CGINS-NUTNRM-00660__20221212.csv (hunk 4 above)
Comment (batryn) [reply calibration/NUTNRM/CGINS-NUTNRM-00660__20221212.csv:8]: spot checked - looks good :)

File: calibration/SPKIRB/CGINS-SPKIRB-00272__20220919.csv (hunk 5 above)
Comment (batryn) [reply calibration/SPKIRB/CGINS-SPKIRB-00272__20220919.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00300__20220919.csv (hunk 6 above)
Comment (batryn) [reply calibration/SPKIRB/CGINS-SPKIRB-00300__20220919.csv:4]: looks good

File: calibration/SPKIRB/CGINS-SPKIRB-00302__20220919.csv (hunk 7 above)
Comment (batryn) [reply calibration/SPKIRB/CGINS-SPKIRB-00302__20220919.csv:4]: looks good :)

**Review by batryn — APPROVED** [review]
looks good - NUTNR CAL spot-checked

**Pull request merged**